tests/test_slow_SW.py

```diff
@@ -0,0 +1,257 @@
+from shutil import rmtree, copyfile
+
+import pytest
+
+import numpy as np
+
+from scipy import stats
+from scipy.signal import argrelmax
+
+from snek5000 import load
+from snek5000_cbox.solver import Simul
+
+
+@pytest.mark.slow
+def params_SW():
+
+    params = Simul.create_default_params()
+
+    aspect_ratio = params.oper.aspect_ratio = 1.0
+    params.prandtl = 0.71
+    params.Ra_side = 1.86e8
+
+    params.output.sub_directory = "tests_snek_cbox"
+
+    params.oper.nproc_min = 2
+    params.oper.dim = 2
+
+    nb_elements = 12
```

Comment:
Could it be run with a bit less elements? It's good to have the tests running as fast as possible.

Reply:
There was a numerical problem. I'll try to fix it with stretching mesh.